components/renku_data_services/utils/core.py

```diff
@@ -0,0 +1,9 @@
+"""Shared utility functions."""
+import ssl
+
+
+def get_ssl_context():
+    """Get an SSL context supporting mounted custom certificates."""
+    context = ssl.create_default_context()
+    context.load_verify_locations(capath="/usr/local/share/ca-certificates")
```

Comment:
```suggestion
    if os.environ.get("SSL_CERT_FILE", None"):
        context.load_verify_locations(cafile=os.environ["SSL_CERT_FILE"])
```

Reply:
Based on this: https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/templates/_certificates-env.tpl

Reply:
Is this actually correct? Going by https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/templates/_certificates-volume-mounts.tpl#L3 and https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/templates/_certificates-volumes.tpl#L11 it could be multiple and the env should just be /etc/ssl/certs/, no?

Reply:
@Panaetius the `"SSL_CERT_FILE"` contains the combination of all SSL certs available in `/etc/ssl/certs/` as well as extra stuff that is added via `/usr/local/share/etc/certs`.

Reply:
For python we only inject the environment variable that points to the cafile that contains all certs. If you want you can also set capath as well. And you can set the capth to always be `/etc/ssl/certs` - regardless of what the evnrionment variable is and that should work. But I think that as you have things right now we will only load any custom/additional certificates from `/usr/local/share/ca-certificates` and completely ignore the system stuff at `etc/ssl/certs`